spec/system/task_spec.rb

```diff
@@ -1,27 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let!(:project) { FactoryBot.create(:project) }
+  let(:task) { FactoryBot.create(:task, project: project)}
```

Comment:
`FactoryBot.`の記載は設定ファイルに追記することで省略できます。
```suggestion
  let(:task) { create(:task, project: project)}
```

Reply:
ec9edd5dd387b4b2253f2af7222d20569c0ea990
`FactoryBot.`の記載を削除しました！